client/rpcserver/rpcserver.go

```diff
@@ -268,16 +268,39 @@ func New(cfg *Config) (*RPCServer, error) {
 	return s, nil
 }
 
-// Run starts the web server. Satisfies the dex.Runner interface.
-func (s *RPCServer) Run(ctx context.Context) {
+type RpcConn interface {
+	Connect(ctx context.Context) (error, *sync.WaitGroup)
+}
+
+func NewRpcConn(s *RPCServer) RpcConn {
+	return s
```

Comment:
If the function is called NewRpcConn, it should construct a new RpcConn. This method doesn't do anything though. It accepts a pointer to an `RPCServer` and returns it.

It is not needed anyway.

Also, I think it is correct to capitalize all of RPC.